### PR TITLE
Changes to expose the v1.1 documentation on kubernetes.io

### PR DESCRIPTION
**Pull request by RichieEscarez**
To be merge for v1.1 GA. This contains pieces to expose the changes in #15893.

You can view these changes in my fork: http://richieescarez.github.io/kubernetes/v1.0/

Note: the version menu and links in the getting started page links dont work properly in the fork due to the addition of the intermediate url path "kubernetes" in the fork's url.

@thockin @janetkuo @mansoorj

### CODE REVIEW

File: _layouts/docwithnav.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
 	<div class="row">
 		<!-- Navigation menu -->
 		<div class="col-md-3 hidden-sm hidden-xs" id="wrapper">
+			{% include docversionselector.html %}
```
Comment (caesarxuchao) [inline _layouts/docwithnav.html]: @nikhiljindal do you know where is the docversionselector.html?

Comment (caesarxuchao) [inline _layouts/docwithnav.html]: Oh, never mind. I misunderstood it.

Comment (caesarxuchao) [inline _layouts/docwithnav.html]: @bgrant0607 @nikhiljindal @janetkuo, maybe we can merge this dropdown menu without other changes in this PR, so the sitemap tool will include v1.1 and users are still guided to v1.0 by default?

Comment (bgrant0607) [inline _layouts/docwithnav.html]: SGTM. Will you create a PR?

Comment (janetkuo) [inline _layouts/docwithnav.html]: :+1:

Comment (janetkuo) [inline _layouts/docwithnav.html]: If sitemap generator still ignore v1.1 links we have this #16606

**Comment by caesarxuchao** [comment]
Just a reminder that we need to merge this when making 1.1 release.

**Pull request merged**